src/content/ui/animations/staggered-animations.md

```diff
@@ -5,101 +5,55 @@ short-title: Staggered
 ---
 
 :::secondary What you'll learn
-* A staggered animation consists of sequential or overlapping
-    animations.
+* A staggered animation consists of sequential or overlapping animations.
 * To create a staggered animation, use multiple `Animation` objects.
-* One `AnimationController` controls all of the `Animation`s.
+* One `AnimationController` controls all of the `Animation` objects.
 * Each `Animation` object specifies the animation during an `Interval`.
 * For each property being animated, create a `Tween`.
 :::
 
-:::tip Terminology
-If the concept of tweens or tweening is new to you, see the
-[Animations in Flutter tutorial][].
+:::tip Prerequisites
+If you don't know about the concept of tweens or tweening,
+consult the [Animations in Flutter tutorial][].
```

Comment:
```suggestion
see the [Animations in Flutter tutorial][].
```

Reply:
Ack. We're using "consult" or "check out" rather than "see" as being more inclusive language.